source/common/grpc/common.h

```diff
@@ -119,10 +120,15 @@ class Common {
                                       std::string* method);
 
   /**
-   * Serialize protobuf message.
+   * Serialize protobuf message. With grpc header.
```

Comment:
nit: with grpc frame header.

How about call this method `serializeToGrpcFrame` and the one without grpc header `serializeMessage`? Having `serializeBody` change its semantic will be problematic for out-of-tree code.

Reply:
Done